tests/transactions/tests.py

```diff
@@ -443,8 +443,25 @@ def test_atomic_does_not_leak_savepoints_on_failure(self):
                 # This is expected to fail because the savepoint no longer exists.
                 connection.savepoint_rollback(sid)
 
-    @skipIf(connection.features.autocommits_when_autocommit_is_off,
-            "This test requires a non-autocommit mode that doesn't autocommit.")
+
+@skipIf(
+    connection.features.autocommits_when_autocommit_is_off,
+    "This test requires a non-autocommit mode that doesn't autocommit."
+)
+class NonAutocommitTests(TransactionTestCase):
+
+    available_apps = []
+
+    def test_orm_query_after_error(self):
```

Comment:
Oops, sorry, I misreported that. Indeed, ValidationError isn't raised until later, but testing with IntegrityError (similar to the test added in tests.py), works in 1.8 but fails in 1.9. Working on investigating with git bisect.